pkg/webhook/sidecarset/validating/sidecarset_create_update_handler.go

```diff
@@ -204,9 +204,10 @@ func (h *SidecarSetCreateUpdateHandler) validateSidecarSetInjectionStrategy(obj
 		}
 
 		switch revisionInfo.Policy {
-		case "", appsv1alpha1.AlwaysSidecarSetInjectRevisionPolicy:
+		case "", appsv1alpha1.AlwaysSidecarSetInjectRevisionPolicy, appsv1alpha1.PartialSidecarSetInjectRevisionPolicy:
 		default:
-			errList = append(errList, field.Invalid(field.NewPath("revision").Child("policy"), revisionInfo, fmt.Sprintf("Invalid policy %v, only support 'Always' currently", revisionInfo.Policy)))
+			errList = append(errList, field.Invalid(field.NewPath("revision").Child("policy"), revisionInfo, fmt.Sprintf("Invalid policy %v, supported: [%s, %s]",
+				revisionInfo.Policy, appsv1alpha1.AlwaysSidecarSetInjectRevisionPolicy, appsv1alpha1.PartialSidecarSetInjectRevisionPolicy)))
```

Comment:
consider add validation for undefined condition for partial strategy
1. setting updateStrategy selector and partition together 
2.  setting updateStrategy partition to integer values

Reply:
setting updateStrategy selector and partition together is not necessary for:
1. if no selector set, all pods will have the chance to be injected the latest revision
2. if no partition set or it's not an integer value (which means partition only works for existing replicas, not for newly created ones), it is considered as 0 in web hook, meaning the chance to inject latest revision is 100%